css/index.css

```diff
@@ -1,3 +1,636 @@
+}
+
+blockquote,
+q {
+    quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+    content: '';
+    content: none;
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+label {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+img {
+    max-width: 100%;
+    max-height: 100%;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+    margin: 30px auto;
+    max-width: 876px;
+}
+
+/* Tablet/Mobile */
+@media (max-width: 800px) {
+    .container {
+        margin: 30px auto;
+        max-width: 730px;
+    }
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* Tablet/Mobile */
+@media(max-width: 800px) {
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+    }
+}
+
+nav {
+    width: 70%;
+    display: flex;
+    justify-content: space-between;
+}
+
+/* Tablet */
+@media(max-width: 800px) {
+    nav {
+        width: 100%;
+        justify-content: space-between;
+        padding: 0 10px;
+    }
+}
+
+/* Mobile */
+@media(max-width: 500px) {
+    nav {
+        display: flex;
+        flex-direction: column;
+        text-align: center;
+        padding: 0;
+    }
+}
+
+header nav a {
+    color: lightgray;
+    text-decoration: none;
+}
+
+/* Mobile Navigation Items */
+@media(max-width: 500px) {
+    header nav a:first-of-type {
+        border-top: 1px solid black;
+    }
+
+    header nav a {
+        border-bottom: 1px solid black;
+        padding: 20px 0;
+    }
+}
+
+/* Tablet Logo */
+@media(max-width: 800px) {
+    .logo {
+        margin-bottom: 40px;
+    }
+}
+
+/* Mobile Log o */
+@media(max-width: 500px) {
+    .logo {
+        margin-bottom: 20px;
+    }
+}
+
+
+.home-content {
+    margin: 80px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    max-width: 100%;
+}
+
+/* Tablet */
+@media(max-width: 800px) {
+    .home-content {
+        display: block;
+        margin: 80px auto;
+    }
+}
+
+/* Mobile */
+@media(max-width: 500px) {
+    .home-content {
+        margin: 40px 0 60px 0;
+    }
+}
+
+
+.home-content div {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    flex-direction: column;
+}
+
+.home-content div h1 {
+    font-size: 90px;
+    text-align: center;
+    margin: 0 auto;
+}
+
+/* Mobile */
+@media(max-width: 500px) {
+    .home-content div h1 {
+        font-size: 60px;
+        text-align: center;
+        margin: 0 auto;
+    }
+}
+
+.home-content div button {
+    margin: 20px auto 0;
+    background: white;
+    border: 1px solid black;
+    height: 35px;
+    width: 200px;
+}
+
```

Comment:
Great job. If you want to go even more responsive, you can refactor your code to use rem instead of pixels.